transcom/mymove
# INT B-22140

B-22140

## Summary
Adding Camp Pendleton, CA as a Counseling Office for LKNQ associated with Camp Pendleton duty location. This is CM'ing a defect fixed in Prod under INC1896099.

Is there anything you would like reviewers to give additional scrutiny?

[this article](tbd) explains more about the approach used.

## Verification Steps for the Author

These are to be checked by the author.


## Verification Steps for Reviewers
When origin duty location is Camp Pendleton, CA, the user should see PPPO DMO Camp Pendleton in the Counseling Office drop down list.

These are to be checked by a reviewer.


### How to test

1. Login as a Service Member
2. Create new orders
3. Select Camp Pendleton, CA as origin duty location
4. Select the Counseling Office drop down and ensure PPPO DMO Camp Pendleton is in the list and can be selected

### Frontend


### Backend


### Database

#### Any new migrations/schema changes:


## Screenshots
